src/components/view/ConnectEhrView/ConnectEhrView.tsx

```diff
@@ -1,17 +1,19 @@
 import React from 'react'
-import {Card, Layout, NavigationBar, StatusBarBackground} from "../.."
+import { Card, Layout, NavigationBar, StatusBarBackground } from "../.."
 import language from '../../../helpers/language'
-import {ProviderSearch} from '../../container'
-import {TextBlock} from '../../presentational'
+import { ProviderSearch, ViewEhr } from '../../container'
+import { TextBlock } from '../../presentational'
 import ExternalAccountsPreview from "../../container/ExternalAccountsPreview";
 
 export interface ConnectEhrViewProps {
-    externalAccountsApplicationUrl: string,
+    externalAccountsApplicationUrl?: string,
+    onViewEhr?: () => void,
     excludeProviders?: boolean,
     excludeHealthPlans?: boolean,
     presentation?: ViewPresentationType,
     preview?: boolean,
     colorScheme?: "auto" | "light" | "dark"
+    introText?: string
 }
 
 export type ViewPresentationType = "Modal" | "Push";
```

Comment:
Should this be exported in index?

Reply:
Nah probably not, I think it does not even need to be a type really